api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "unversioned"
+   "version": "v1.6.0"
```

Comment:
I think you already went looking for him, but just in case: @david-mcmahon may have suggestions about this.